kale/worker.py

```diff
@@ -364,3 +364,9 @@ def _run_batch(self, message_batch):
 
             # Increment total messages counter.
             self._total_messages_processed += 1
+
+    def run_task(self, message):
```

Comment:
This is so that you can subclass `Worker` and add worker-level logging.  As opposed to right now were you can only to task-level logging?